scripts/stratis_udev/stratis_uuids_to_names.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python3
```

Comment:
Right now the ```scripts``` directory is holding one Python script that we would not choose to distribute. But this one we would. So, either this should go somewhere else, or more likely the existing script should be moved somewhere else (or possibly removed).

Reply:
We would want to run all our usual linting and formatting facilities on this script once we decided where to put it. I ran the linter and saw a couple actually interesting errors, as well as some minor things. I'm just going to do a high-level review for right now.